src/ape/api/accounts.py

```diff
@@ -195,10 +195,9 @@ def deploy(self, contract: "ContractContainer", *args, **kwargs) -> "ContractIns
             raise AccountsError(f"'{receipt.txn_hash}' did not create a contract.")
 
         address = click.style(receipt.contract_address, bold=True)
-        logger.success(f"Contract '{contract.contract_type.contractName}' deployed to: {address}")
+        logger.success(f"Contract '{contract.contract_type.name}' deployed to: {address}")
```

Comment:
should this still be `contractName`?
Yesterday, I thought we saw that or something

Reply:
We investigated this through compiling and deploying a contract and the `.name` property is the correct naming.